### PR TITLE
Remove v2 from documentation

### PR DESCRIPTION
**Pull request by seelchen**
This removes all Codyze v2 specific pages and parts from the documentation.

### CODE REVIEW

File: docs/Getting Started/configuration.md [hunk 1]
```diff
@@ -58,26 +50,26 @@ The configuration structure for __Codyze v3__ separates the options by subcomman
...
 
 ## List of Configurations
-This is a list of all available configurations, their descriptions and their respective name in __Codyze v2__ and __Codyze v3__.
+This is a list of all available configurations, their descriptions and their respective name.
 The names are the same for the configuration file and the CLI options.
 
 `./` denotes the working directory in which Codyze was started.
 
-| Codyze v2               | Codyze v3               | Value               | Description                                                                                                                                                                                                                                               | Default Value     |
+| Legacy Codyze           | Redesigned Codyze       | Value               | Description                                                                                                                                                                                                                                               | Default Value     |
```
Comment (fwendland) [inline docs/Getting Started/configuration.md:61]: We can remove the whole first column with legacy Codyze.

File: docs/Getting Started/configuration.md [hunk 2]
```diff
@@ -12,40 +12,32 @@ There are two ways of configuring Codyze, through command line options or a conf
...
 * Command line mode:
   <br />Non-interactive command line client, accepts arguments from command line and runs analysis
 * Language server protocol mode:
   <br />This mode is for IDE support and binds to stdout as a server for Language Server Protocol (LSP)
 * Interactive console mode:
   <br />The text based user interface (TUI) is an interactive console that allows exploring the analyzed source code by manual queries
 
-The execution mode is specified in __Codyze v2__ through an option flag (`-c`, `-l` or `-i`).
-For __Codyze v3__ subcommands are used (`analyze`, `lsp`, or `interactive`).
+Subcommands are used to enter the modes (`analyze`, `lsp`, or `interactive`).
```
Comment (fwendland) [inline docs/Getting Started/configuration.md:28]: Are we still supporting all subcommands?

I remember that we decided to support only `analyze` fot now. @maximilian-galanis can you confirm?

File: docs/Getting Started/cli.md [hunk 3]
```diff
@@ -66,12 +56,8 @@ jobs:
...
 Below are the commands to call Codyze on these examples.
 
-```shell title="Codyze v2"
-./gradlew :codyze-v2:run --args="-c -s ../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp -m ../examples/botan/MARK"
-```
-
-```shell title="Codyze v3"
-./gradlew :codyze-v3:codyze:run --args="analyze -s ../../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp --spec ../../examples/botan/MARK"
+```shell
+./gradlew :codyze-cli:run --args="analyze -s ../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp --spec ../examples/botan/MARK"
```
Comment (fwendland) [inline docs/Getting Started/cli.md:60]: I would remove this example as we do not have MARK support in v3.

We could instead refer to our test resources, maybe.

**Review by fwendland — CHANGES_REQUESTED** [review]

File: docs/Getting Started/configuration.md (hunk 1 above)
Comment (seelchen) [reply docs/Getting Started/configuration.md]: Done in 836e36ac0f3a919b50b132a1e16f102b3dd951a1.

File: docs/Getting Started/configuration.md (hunk 2 above)
Comment (seelchen) [reply docs/Getting Started/configuration.md:28]: I think we should change that in #671 where the subcommands are removed.

File: docs/Getting Started/cli.md (hunk 3 above)
Comment (seelchen) [reply docs/Getting Started/cli.md]: Done in f1738c413119693e02d2122ea98faa8bc277fc27.

**Review by fwendland — APPROVED** [review]

**Pull request merged**